src/components/SortControl.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+
+const SortControl = ({selectedSorting, onChange}) => {
+    const handleOnChange = (e) => {
+        console.log(typeof onChange)
+        if (typeof onChange === "function") {
```

Comment:
To avoid such check better to use static code analyzer
Lets use Typescript starting from next module

Reply:
> To avoid such check better to use static code analyzer Lets use Typescript starting from next module

this condition is used for storybook, as it receives the function for event from parent component, but in storybook it does not have function for this